doc/conf.py

```diff
@@ -19,7 +19,7 @@
 supported_confluence_ver = '7.19+'
 supported_python_ver = '3.8+'
 supported_requests_ver = '2.25.0+'
-supported_sphinx_ver = '6.1+'
+supported_sphinx_ver = '7.1+'
```

Comment:
@jdknight Hi, the `pyproject.toml` still requires `Sphinx>=6.1`, which can break installations, as `sphinxcontrib-confluencebuilder` is upgrade to `2.7` without bumping `Sphinx` to 7.1.

See the error below.

```
Sphinx version error:
The sphinxcontrib.confluencebuilder extension used by this project needs at least Sphinx v7.1; it therefore cannot be built with this version.
```

Reply:
@robert-schmidtke, thanks for the report. Should be addressed in the v2.7.1 release.

Reply:
Thanks for resolving this so quickly!